pkg/apis/certificates/helpers.go

```diff
@@ -87,16 +93,22 @@ func ValidateKubeletServingCSR(req *x509.CertificateRequest, usages sets.String)
 	return nil
 }
 
-var kubeletClientRequiredUsages = sets.NewString(
-	string(UsageDigitalSignature),
-	string(UsageKeyEncipherment),
-	string(UsageClientAuth),
+var (
+	kubeletClientRequiredUsagesNoRSA = sets.NewString(
+		string(UsageDigitalSignature),
+		string(UsageClientAuth),
+	)
+	kubeletClientRequiredUsages = sets.NewString(
+		string(UsageDigitalSignature),
+		string(UsageKeyEncipherment),
+		string(UsageClientAuth),
+	)
 )
 
 func IsKubeletClientCSR(req *x509.CertificateRequest, usages sets.String) bool {
```

Comment:
this is called by v1beta1.IsKubeletClientCSR, so it need to take the allowOmittingUsageKeyEncipherment param as well, not hard-code to true here

Reply:
Fixed.